docs/Installation/get_started.md

```diff
@@ -0,0 +1,463 @@
+# Get Started
+## Prerequisite
+* Mac OS X 10.7 (Lion) or newer or a linux OS (tested on ubuntu 18.04)
+* Git installed with a github account
+* an [ssh key](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) added to [github](https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account)
+    * to see if SSL key has been loaded `ssh-add -L` 
```

Comment:
`ssh-add -L`  show the ssh key loaded in an ssh-agent. This has nothing to do with github. The way it is formulated here suggest you can check if you have a key added to github `with ssh-add -L`